packages/trace-viewer/src/progress.ts

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-type Progress = (done: number, total: number) => void;
+type Progress = (done: number, total: number) => undefined;
```

Comment:
This looks very odd. Why do we need to change it to undefined? Event in zip.js it is [Promise<void>](https://github.com/gildas-lormeau/zip.js/blob/e140faab662147bb181fb5f4a2b8ac67de9fd416/index.d.ts#L1228C4-L1228C75) I believe.